### DISCUSSION TITLE
DIM Plus v 1.0 Feedback

### DISCUSSION THREAD

**Discussion by mgibbonsjr** [leonardoneumann/DimPlus#30] [General]
Wasn't sure of the best way to give feedback so hopefully this is ok. 

1. The top perks in the panel on the right is a nice touch and loads extremely fast. Kudos on that. 
2. I really liked the feature that would pull up a side menu next to the gun to show the top trait combos and highlight the combo in that list if the gun matched it. Version 1.0 seems as though this feature is missing or maybe still in development. Originally I uninstalled version 0.9 to replace with 1.0, but I then installed 0.9 again alongside 1.0 and now have both features working. 

Thanks again for all of your hard work!

**Comment by leonardoneumann** [comment]
Thanks for your feedback ! I'll add combos, masterwork and missing intrinsic perks data in the next days

**Comment by leonardoneumann** [comment]
trait combos , masterwork and mods available in the newest release, still need to highlight the combo. Will do so in the next days

**Comment by leonardoneumann** [comment]
Added the higlight feature for perk combos on release 1.0.2

**Reply by mgibbonsjr** [reply]
You are the best!!!! Thank you so much for the hard work. I know this stuff is time consuming and no financial gain for you, so I'll donate some so you can at least have some coffee to help :) 

Thanks again bud!

**Reply by leonardoneumann** [reply]
Thanks a lot for your support ! I'll keep updates coming when I get time to do so.
I'm aware of some issues with new weapons and usabilty problems. Will also implement the 'compare' feature soon.
